lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-redhat-infra_manager.rb

```diff
@@ -2,5 +2,6 @@ module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Redhat_InfraManager < MiqAeServiceEmsInfra
     expose :validate_import_vm
     expose :submit_import_vm
+    expose :submit_configure_imported_vm_networks
```

Comment:
the logic here is that the `submit_import_vm` actually submits the method call as a task via the queue on the provider object itself. I felt like I should obey this semantic also in the configure networks case.

And yes it is used only to "fix" the networks after the VM has been imported using the v2v process. So this is all expressed in the name. Although I agree it is a bit mouthful.